samples/cpp/arrow/makefile_lnx

```diff
@@ -50,19 +50,19 @@ ifndef compiler
 endif
 
 ifndef DAALROOT
-    DAALROOT = ./../..
+    DAALROOT = ./../../..
 endif
 
 ifndef TBBROOT
     TBBROOT = ./../../../../tbb/latest
 endif
-TBB_PATH = "$(TBBROOT)/lib/$(_IA)/gcc4.8"
+TBB_PATH = $(TBBROOT)/lib/$(_IA)/gcc4.8
```

Comment:
Revert quotes (at least to be consistent with other samples).

Reply:
Revert was done for both comments.